lib/widgets/autocomplete.dart

```diff
@@ -0,0 +1,257 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Add a comment here saying where this code is from and why copying it was necessary.